### ISSUE TITLE
Run Blender button opens blender w/o drag-n-drop functionality

### ISSUE THREAD

**Issue by EdwHopper** [overmindstudios/BlenderUpdater#45]
A small issue is that opening Blender from the interface button after finishing the update seems to disable drag and drop for oppening scenes or importing images. Just re/starting from the usual shortcut works fine.

**Comment by tobkum** [comment]
Ohh, interesting - what operating system are you using? Windows?

**Comment by EdwHopper** [comment]
Yes, Windows 10. It may be just for me as I was unable to update it for more than 6 months?

**Comment by tobkum** [comment]
Hm - you are using the latest version of BlenderUpdater, right? The one that came out a few days ago?

**Comment by EdwHopper** [comment]
Yes, 1.9.10, but it was the same with the last 2 previous versions.

**Comment by tobkum** [comment]
I'll have a look on the weekend - thanks for reporting!
